src/Raven.Server/Documents/Replication/ResolveConflictOnReplicationConfigurationChange.cs

```diff
@@ -400,6 +406,57 @@ public DocumentConflict ResolveToLatest(List<DocumentConflict> conflicts)
 
             return latestDoc;
         }
+
+        private void ResolveAttachmentsConflicts(DocumentsOperationContext context, DocumentConflict resolved)
+        {
+            using var scope = Slice.External(context.Allocator, resolved.LowerId, out var lowerId);
+            var storageAttachmentsDetails = _database.DocumentsStorage.AttachmentsStorage.GetAttachmentDetailsForDocument(context, lowerId);
+            List<AttachmentName> resolvedAttachmentsMetadata = null;
+            bool resolveToLatest = ScriptConflictResolversCache.TryGetValue(resolved.Collection, out ScriptResolver scriptResolver) == false || scriptResolver == null;
```

Comment:
Note that you can define both `ResolveToLatest` and a script.
In that case, we try the script first and if we not succeed we will resolve to latest